.babelrc

```diff
@@ -1,8 +1,10 @@
 {
   "presets": [
+    "@babel/preset-flow",
+    "@babel/preset-react",
     [
-      "react-app",
-      { "absoluteRuntime": false, "flow": true, "typescript": false }
+      "@babel/preset-env",
+      {"targets": "last 1 Chrome version, last 1 Firefox versions"}
```

Comment:
Why do we need to set `targets` to the newest browsers? Can't we just set it to something like 
`"chrome": "50","firefox": "69"`

Reply:
Ah for this I figured we could only support the latest and greatest since we're targeting web developers